pandas/io/tests/test_pickle.py

```diff
@@ -32,6 +33,7 @@ class TestPickle():
     """
     _multiprocess_can_split_ = True
 
+    @pytest.fixture(autouse=True)
```

Comment:
I think this might have been my suggestion. 

Without this, I'm not sure `setUp` will run unless the class inherits from `unittest.TestCase`.

I think we could change `setUp` to `[setup_class](http://doc.pytest.org/en/latest/xunit_setup.html)` and it would run `pytest`. (Or `setup_method` if we need access to `self` rather than `class`)